File: iPhone/VerBank-CSTSv3-IOSClientApi/CSTS3Comm/comm/FnCertState.m
//
//  FnCertState.m
//  VerBank-IOSClientStation
//
//  Created by Allone on 15/10/21.
//  Copyright © 2015年 zhanglei. All rights reserved.
//

#import "FnCertState.h"

static NSString * jsonId = @"FnCertState";

static NSString * returnCode            = @"1";
static NSString * requestID             = @"2";
static NSString * caSerial              = @"3";
static NSString * caState               = @"4";
static NSString * cn                    = @"5";
static NSString * beginValidTime        = @"6";
static NSString * endValidTime          = @"7";

@implementation FnCertState

- (id)init {
    if(self = [super init]) {
        [super setEntry:@"jsonId" entry:jsonId];
    }
    return self;
}

jsonImplementionInt(ReturnCode,             returnCode)
jsonImplementionInt(RequestID,              requestID)
jsonImplementionString(CaSerial,            caSerial)
jsonImplementionInt(CaState,                caState)
jsonImplementionString(Cn,                  cn)
jsonImplementionLongLong(BeginValidTime,    beginValidTime)
jsonImplementionLongLong(EndValidTime,      endValidTime)

@end

File: WeChatService/VerBank-Ctrl/src/allone/MTP/VerBank01/Ctrl/comm/structure/FnCertState.java
package allone.MTP.VerBank01.Ctrl.comm.structure;

import java.io.Serializable;

/**
 * 
 * @Project: VerBank-Ctrl 
 * @Title: FnCertState.java 
 * @Package allone.MTP.VerBank01.Ctrl.comm.structure 
 * @Description: 取得憑證　(QueryCert) Retun code|憑證申請 ID|憑證序號|憑證狀態代碼|CN|憑證效期起|憑證效期迄
 * @author wyb wang.yubo@allone.cn 
 * @date 2015年10月15日 下午4:47:05 
 * @Copyright: 2015 www.allone.cn Inc. All rights reserved. 
 * @version V3.0.0   
 */
public class FnCertState implements Serializable {

	private static final long serialVersionUID = -6071049073830642864L;

	private int returnCode;

	// 憑證申請 ID
	private int requestID;
	// 憑證序號
	private String caSerial;
	// 憑證狀態代碼
	private int caState;

	private String cn;

	// 憑證效期起
	private long beginValidTime;

	// 憑證效期迄
	private long endValidTime;

	public int getReturnCode() {
		return returnCode;
	}

	public void setReturnCode(int returnCode) {
		this.returnCode = returnCode;
	}

	public int getRequestID() {
		return requestID;
	}

	public void setRequestID(int requestID) {
		this.requestID = requestID;
	}

	public String getCaSerial() {
		return caSerial;
	}

	public void setCaSerial(String caSerial) {
		this.caSerial = caSerial;
	}

	public int getCaState() {
		return caState;
	}

	public void setCaState(int caState) {
		this.caState = caState;
	}

	public String getCn() {
		return cn;
	}

	public void setCn(String cn) {
		this.cn = cn;
	}

	public long getBeginValidTime() {
		return beginValidTime;
	}

	public void setBeginValidTime(long beginValidTime) {
		this.beginValidTime = beginValidTime;
	}

	public long getEndValidTime() {
		return endValidTime;
	}

	public void setEndValidTime(long endValidTime) {
		this.endValidTime = endValidTime;
	}

}
